examples/04_functional/01_Higher-Order Functions.md

```diff
@@ -39,7 +39,7 @@ fun main() {
 }
 ```
 
-1. Declares a higher-order function that returns a function.
+1. Declares a higher-order function that returns a function. (Int) -> Int represents the parameters and return type of square function.
```

Comment:
```suggestion
1. Declares a higher-order function that returns a function. `(Int) -> Int` represents the parameters and return type of the `square` function.
```

Reply:
Thanks! Please format the code elements in the text as suggested and we'll merge this.